var/spack/repos/builtin/packages/fzf/package.py

```diff
@@ -38,6 +42,11 @@ def determine_version(cls, exe):
         match = re.match(r"(^[\d.]+)", output)
         return match.group(1) if match else None
 
+    def url_for_version(self, version):
+        if self.spec.satisfies("@:0.53.0"):
+            return f"https://github.com/junegunn/fzf/archive/{version}.tar.gz"
+        return f"https://github.com/junegunn/fzf/archive/v{version}.tar.gz"
```

Comment:
FYI. The following change allows me to (re)confirm all of the version sha256s for the package and tighten the code just a tiny bit:

```suggestion
        base = "refs/tags/v" if self.spec.satisfies("@:0.53.0") else ""
        return f"https://github.com/junegunn/fzf/archive/{base}{version}.tar.gz"
```